.github/workflows/module-test.yml

```diff
@@ -9,8 +9,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target: [ubuntu-18.04, ubuntu-20.04, ubuntu-22.04, debian-10, debian-11]
+        target:
+          [
+            { os: ubuntu, version: 20.04, package-type: DEB, excluded-tests: "SecurityBaseline|Sample" },
```

Comment:
Why is SecurityBaseline excluded?

Reply:
Its broken and fails on every distro. Its been excluded for now.

Reply:
Is this being tracked? What is required to fix it?

Reply:
Yes, this will be part of my summary email to the team about the current state/gaps, etc.